contrib/mesos/pkg/scheduler/plugin.go

```diff
@@ -437,7 +437,7 @@ func (q *queuer) Run(done <-chan struct{}) {
 			}
 
 			pod := p.(*Pod)
-			if recoverAssignedSlave(pod.Pod) != "" {
+			if recoverAssignedSlave(pod.Pod) != "" || pod.InGracefulTermination() {
```

Comment:
remind me why we need this here.. how is it possible for a pod to be in graceful termination w/o having been assigned a binding annotation?